geometry_msgs/Point position
geometry_msgs/Point dimension
